### PR TITLE
Yousef Katat

### PR DESCRIPTION
**Pull request by YousefKatat**

### CODE REVIEW

File: Week1/exercise/w1/index.js [hunk 1]
```diff
@@ -14,6 +14,15 @@ console.log('header: ', header);
 //    log input value
 //    set image src to imageInput element value
 
+const changeImageButton = document.querySelector('#btn-changeImage')
+const changeImage = () => {
+    const image = document.getElementById("imageToChange");
+    const imageInput = document.getElementById("imageInput");
```
Comment (ldealmei) [inline Week1/exercise/w1/index.js:20]: Select elements only once, and outside the function

File: Week1/exercise/w1/index.js [hunk 2]
```diff
@@ -14,6 +14,15 @@ console.log('header: ', header);
...
+    image.src = 'https://www.nieuwsbladtransport.nl/wp-content/uploads/2019/03/big-data-cyber-security-hack.jpg';
```
Comment (ldealmei) [inline Week1/exercise/w1/index.js:21]: Try now, to change `image.src` to the value of the `imageInput`

File: Week1/exercise/w1/index.js [hunk 3]
```diff
@@ -28,3 +37,15 @@ console.log('header: ', header);
 //    create <li> element
 //    set <li> element innerHtml to todoInput value
 //    add <li> element to todoList
+
+const addTodoButton = document.querySelector("#btn-addTodo");
+const addTodo = () => {
+
+    const todoList = document.getElementById("todoList");
+    const todoInput = document.getElementById("todoInput");
```
Comment (ldealmei) [inline Week1/exercise/w1/index.js:45]: Idem, try selecting elements outside of the function, so that you don't select multiple times the same element

**Review by ldealmei — CHANGES_REQUESTED** [review]

**Review by bitsagarob — APPROVED** [review]
great job, try to implement the remarks though

**Comment by YousefKatat** [comment]
week 2 - Yousef

**Comment by YousefKatat** [comment]
week 3

**Pull request closed without merging**